Sources/SwiftJavaRuntimeSupport/JNI.swift

```diff
@@ -15,13 +15,28 @@
 import SwiftJava
 import CSwiftJavaJNI
 
-final class JNI {
-  static var shared: JNI!
+public final class JNI {
+  public fileprivate(set) static var shared: JNI!
 
-  let applicationClassLoader: JavaClassLoader
+  public let applicationClassLoader: JavaClassLoader
+  public let globalArena: JavaSwiftArena
```

Comment:
Hmm, this makes me realize the naming of the constant is wrong as well. "[Global](https://docs.oracle.com/en/java/javase/24/docs/api/java.base/java/lang/foreign/Arena.html#global())" arena is specifically an arena that does not track references and things are just immortal, this isn't a global arena, this is an "[auto arena](https://docs.oracle.com/en/java/javase/24/docs/api/java.base/java/lang/foreign/Arena.html#ofAuto())".

Perhaps the `GLOBAL_SWIFT_JAVA_ARENA` should become `DEFAULT_SWIFT_JAVA_AUTO_ARENA` perhaps. If easy enough, we can get rid of the constant as well; just upcall to get the `SwiftArena.ofAuto()` here, but I don't mind us keeping the constant if it makes our lives much easier.

Reply:
I still think we want to use the constant, as this is the arena that is also used when you enable `allowGlobalArena` in `memoryManagementMode`. I think it makes sense to just always use the same one, whenever we need to reach for a "global" one? 

But sure, I think `DEFAULT` would be better in terms of naming